src/main/java/com/indeed/jiraactions/jiraissues/JiraIssuesParser.java

```diff
@@ -0,0 +1,80 @@
+package com.indeed.jiraactions.jiraissues;
+
+import com.google.common.base.Stopwatch;
+import com.univocity.parsers.tsv.TsvParser;
+import com.univocity.parsers.tsv.TsvParserSettings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+public class JiraIssuesParser {
+    private static final Logger log = LoggerFactory.getLogger(JiraIssuesIndexBuilder.class);
+    private static TsvParserSettings settings = setupSettings(new TsvParserSettings());
+
+    private final JiraIssuesProcess process;
+    private final JiraIssuesFileWriter fileWriter;
+
+    private final List<String[]> newIssues;
+    private File file;
+    private FileReader reader;
+    private TsvParser parser;
+
+    public JiraIssuesParser(final JiraIssuesFileWriter fileWriter, final JiraIssuesProcess process, final List<String[]> newIssues) {
+        this.fileWriter = fileWriter;
+        this.process = process;
+        this.newIssues = newIssues;
+    }
+
+    private static TsvParserSettings setupSettings(final TsvParserSettings settings) {
```

Comment:
Why pass in settings instead of just instantiating the default one inside the method?